internal/fdo.go

```diff
@@ -118,7 +119,7 @@ func fdoLookupApplicationByMetadata(appName string) desktop.AppData {
 }
 
 //fdoLookupApplication looks up an application by name and returns an fdoApplicationData struct
-func fdoLookupApplication(appName string) desktop.AppData {
+func fdoLookupApplication(appName string, fallbackIfMissing bool) desktop.AppData {
```

Comment:
Every single call to this is `, false)` except once.
Therefore it would be neater to return it from the fdoLookupApplicationWinInfo func as a last resort.
Passing bool flags is usually a code smell...